package.json

```diff
@@ -43,14 +47,27 @@
     "command-line-args": "^5.1.1",
     "dotenv": "^10.0.0",
     "eslint": "^7.29.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-import": "^2.23.4",
+    "eslint-plugin-prefer-arrow": "^1.2.3",
+    "eslint-plugin-prettier": "^3.4.0",
+    "husky": "^7.0.0",
     "jest": "^26.6.3",
     "jest-junit-reporter": "^1.1.0",
+    "lint-staged": "^11.0.1",
     "nock": "^13.1.0",
     "nodemon": "^2.0.7",
+    "prettier": "^2.3.2",
     "sprintf-js": "^1.1.2",
     "supertest": "^6.1.3",
     "ts-jest": "^26.5.6",
     "ts-node": "^10.0.0",
     "typescript": "^4.3.2"
+  },
+  "lint-staged": {
```

Comment:
it's back again lol

Reply:
This part we actually need—it was the `husky: {}` that we didn't need.